add substrate-connect tutorial (#1122)
substrate-developer-hub/substrate-docs: content/md/en/docs/tutorials/light-clients/parachains.md

Go ahead and save it into the same directory from the previous tutorial (`empty-webapp`). Then, edit the contents of `index.ts` so it looks like the following:
```typescript
import {
  ScProvider,
  WellKnownChain,
} from "@polkadot/rpc-provider/substrate-connect";
import { ApiPromise } from "@polkadot/api";
import jsonParachainSpec from "./statemint.json";

window.onload = () => {
  void (async () => {
    try {
      const relayProvider = new ScProvider(WellKnownChain.polkadot);
      const parachainSpec = JSON.stringify(jsonParachainSpec);
      const provider = new ScProvider(parachainSpec, relayProvider);

      await provider.connect();
      const api = await ApiPromise.create({ provider });
      await api.rpc.chain.subscribeNewHeads(
        (lastHeader: { number: unknown; hash: unknown }) => {
          console.log(
            `New block #${lastHeader.number} has hash ${lastHeader.hash}`
          );
        }
      );
    } catch (error) {
      console.error(<Error>error);
    }
  })();
};
```

This code is very similar to the previous one. The main differences are

> - we are loading `statemint.json` into a `parachainSpec` object.

Reviewer: ```suggestion
- we are loading `statemint.json` into the `parachainSpec` variable.
```

> - we are creating a `relayProvider` object for the Relay chain, and using it in the construction of the `provider` object for the Parachain, along with `parachainSpec`.

Reviewer: ```suggestion
- `ScProvider` is created for relay chain but without connecting (since we will not be communicating with the relay chain). This is needed as a parameter when creating the Parachain's provider as an extra parameter. Rpc Provider of substrate-connect need this, in order to connect the parachain to the respective relay chain.
```

---

> title: Connect with a parachain

Reviewer 2: gatsby  mangles things with the same file name - can this file be renamed to to something less generic than `parachains.md`?

Author: I initially tried `parachain.md` and indeed there was some crazy behavior.
Maybe `connect-with-a-parachain.md`, together with `connect-with-a-relay-chain.md`?
